### PR TITLE
taxonomy: Category translations

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: Category translations

### CODE REVIEW

**Comment by teolemon** [comment]
ERROR - nl:yoghurts-met-fruit-en-suiker already is associated to en:yogurt-with-fruits-and-sugar (categories) - nl:yoghurts-met-fruit-en-suiker cannot be mapped to entry en:yogurts-with-fruits-and-sugar
ERROR - fi:kanavoileivät already is associated to en:poultry-sandwiches (categories) - fi:kanavoileivät cannot be mapped to entry en:chicken-sandwiches

**Comment by alexgarel** [comment]
@teolemon 

```
Errors in the categories taxonomy definition:
ERROR - nl:yoghurts-met-fruit-en-suiker already is associated to en:yogurt-with-fruits-and-sugar (categories) - nl:yoghurts-met-fruit-en-suiker cannot be mapped to entry en:yogurts-with-fruits-and-sugar
ERROR - fi:kanavoileivät already is associated to en:poultry-sandwiches (categories) - fi:kanavoileivät cannot be mapped to entry en:chicken-sandwiches
```

**Comment by teolemon** [comment]
yes, I commented that 2hours ago ;-)

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -88468,13 +88973,20 @@ de:Geflügelsandwiches
 fi:siipikarjavoileivät
 fr:Sandwichs à la volaille
 nl:Gevogelte sandwiches
+es:Bocadillos de aves
+it:Panini di carne avicola
+pt:Sanduíches de aves
+bg:Сандвичи с птици
+es:Sándwiches de aves
+it:Panini al pollame
+nl:Pluimvee sandwiches
```
Comment (benbenben2) [inline taxonomies/categories.txt:88982]:
```suggestion
nl:Gevogelte sandwiches, pluimvee sandwiches
es:Bocadillos de aves
it:Panini di carne avicola
pt:Sanduíches de aves
bg:Сандвичи с птици
es:Sándwiches de aves
it:Panini al pollame
```

**Review by benbenben2 — APPROVED** [review]
LGTM

found synonyms. See suggestion

**Comment by teolemon** [comment]
ERROR - nl:dolce-gusto-compatibele-koffiecapsules already is associated to en:dolce-gusto-compatible-coffee-capsules (categories) - nl:dolce-gusto-compatibele-koffiecapsules cannot be mapped to entry en:dolce-gusto-compatible-capsules
ERROR - pt:capsulas-de-cafe-compativeis-com-dolce-gusto already is associated to en:dolce-gusto-compatible-coffee-capsules (categories) - pt:capsulas-de-cafe-compativeis-com-dolce-gusto cannot be mapped to entry en:dolce-gusto-compatible-capsules

**Comment by teolemon** [comment]
@benbenben2 I did a substantial commit from another branch on my laptop after your review. Could you have a look at that commit ?
https://github.com/openfoodfacts/openfoodfacts-server/pull/9673/commits/622824a0c4d2c0f04956101337b5ddd9bbeae4b3

If everything still ok, please tell me so and I merge.

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -49575,6 +50196,16 @@ agribalyse_proxy_food_name:fr:Yaourt, lait fermenté ou spécialité laitière,
 #wikidata:en:
 intake24_category_code:en:FTYG
 
+<en:Fruit yogurts
+fr:Yaourts sucrés aux fruits
```
Comment (benbenben2) [inline taxonomies/categories.txt:50200]: Let's keep the English translation
```suggestion
en:Yogurts with fruits and sugar
fr:Yaourts sucrés aux fruits
```

**Review by benbenben2 — APPROVED** [review]
LGTM, one comment about English translation.

**Pull request merged**